backend/question.py

```diff
@@ -6,3 +6,73 @@
 @question_page.route('/question/ping', methods=['GET'])
 def question_ping():
     return "Pong", 200
+
+# allow user to post a question and word account not more than 200 words
+@question_page.route('/question/add', methods=['POST'])
+def question_add():
+    data = request.get_json()
+    user_id = data['user_id']
+    question = data['question']
+    answer = data['answer']
+    if len(question) > 200:
+        return make_response(jsonify({"error": "Question is too long"})), 400
+    if len(answer) > 200:
+        return make_response(jsonify({"error": "Answer is too long"})), 400
+    con = sqlite3.connect(DATABASE_NAME)
+    cur = con.cursor()
+    sql = "insert into questions values (?, ?, ?, ?)"
+    cur.execute(sql, [None, user_id, question, answer])
+    con.commit()
+    return make_response(jsonify({"success": "Question added"})), 200
+
+
+# allow user to get all questions
+@question_page.route('/question/get', methods=['GET'])
```

Comment:
most likely we will have a question_id to get instead of getting all the questions. This will be similar to 
`@article_page.route('/article/<int:article_id>', methods=['GET'])` in article.py